ucsb-cs156-f24/proj-happycows-f24-10
# Add additional web integration test for other Commons functions

## Overview

In this PR, I add a new web integration test that checks whether an admin user can edit/delete a commons.

## Feedback Request

Please give suggestions on the following:
- Should I be more thorough when it comes to editing? I just update the name and check that it updates, and leave everything else unchanged.

## Validation

1. Download this branch.
2. Add admingaucho@ucsb.edu to ADMIN_EMAILS in your `.env` file
3. Run `INTEGRATION=true mvn test-compile` and then `INTEGRATION=true mvn failsafe:integration-test`, and check that the test suite passes.

## Tests


## Linked Issues

Closes #57
